models/agent_model.go

```diff
@@ -73,10 +73,13 @@ type Agent struct {
 	ListenPort *uint16 `reform:"listen_port"`
 	Version    *string `reform:"version"`
 
-	Username      *string `reform:"username"`
-	Password      *string `reform:"password"`
-	TLS           bool    `reform:"tls"`
-	TLSSkipVerify bool    `reform:"tls_skip_verify"`
+	Username                      *string `reform:"username"`
+	Password                      *string `reform:"password"`
+	TLS                           bool    `reform:"tls"`
+	TLSSkipVerify                 bool    `reform:"tls_skip_verify"`
+	TLSCertificateKey             string  `reform:"tls_certificate_key"`
+	TLSCertificateKeyFilePassword string  `reform:"tls_certificate_key_file_password"`
+	TLSCaKey                      string  `reform:"tls_ca_key"`
```

Comment:
@JiriCtvrtka @BupycHuk I think it might be a time for us to start adding those new properties into a single JSON column

Reply:
yeah, it might be good reason for that and probably we will need one more task to move other additional fields to single json column.
Other question is should we use the same column for all fields or split them by groups? Using separate columns for each service type will be easier to unmarshal and keep only required field. what do you think?

Reply:
> separate columns for each service type

Sounds good